decidim-accountability/app/views/decidim/accountability/admin/results/index.html.erb

```diff
@@ -22,102 +22,100 @@
+              <% end %>
 
-                <% if allowed_to? :update, :result, result: result %>
-                    <%= icon_link_to "folder", result_attachment_collections_path(result), t("actions.attachment_collections", scope: "decidim.accountability"), class: "action-icon--attachment_collections" %>
-                <% end %>
+              <% if allowed_to? :update, :result, result: result %>
+                  <%= icon_link_to "folder-line", result_attachment_collections_path(result), t("actions.attachment_collections", scope: "decidim.accountability"), class: "action-icon--attachment_collections" %>
+              <% end %>
 
-                <% if allowed_to? :update, :result, result: result %>
-                  <%= icon_link_to "paperclip", result_attachments_path(result), t("actions.attachments", scope: "decidim.accountability"), class: "action-icon--attachments" %>
-                <% end %>
+              <% if allowed_to? :update, :result, result: result %>
+                <%= icon_link_to "attachment-line", result_attachments_path(result), t("actions.attachments", scope: "decidim.accountability"), class: "action-icon--attachments" %>
+              <% end %>
 
-                <%= icon_link_to "eye", resource_locator(result).path, t("actions.preview", scope: "decidim.accountability"), class: "action-icon--preview", target: :blank %>
+              <%= icon_link_to "eye-line", resource_locator(result).path, t("actions.preview", scope: "decidim.accountability"), class: "action-icon--preview", target: :blank, data: { "external-link": false } %>
 
-                <%= resource_permissions_link(result) %>
+              <%= resource_permissions_link(result) %>
 
-                <% if allowed_to? :destroy, :result, result: result %>
-                  <%= icon_link_to "circle-x", result_path(result), t("actions.destroy", scope: "decidim.accountability"), class: "action-icon--remove", method: :delete, data: { confirm: t("actions.confirm_destroy", scope: "decidim.accountability", name: t("models.result.name", scope: "decidim.accountability.admin")) } %>
-                <% end %>
-              </td>
-            </tr>
-          <% end %>
-        </tbody>
-      </table>
-      <%= paginate results, theme: "decidim" %>
-    </div>
+              <% if allowed_to? :destroy, :result, result: result %>
+                <%= icon_link_to "delete-bin-line", result_path(result), t("actions.destroy", scope: "decidim.accountability"), class: "action-icon--remove", method: :delete, data: { confirm: t("actions.confirm_destroy", scope: "decidim.accountability", name: t("models.result.name", scope: "decidim.accountability.admin")) } %>
+              <% end %>
+            </td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
   </div>
 </div>
+<%= decidim_paginate results, theme: "decidim" %>
```

Comment:
`decidim_paginate` is already setting the theme argument, and setting there does not have the expected outcome.

```suggestion
<%= decidim_paginate results %>
```